agenta-backend/agenta_backend/routers/app_evaluation_router.py

```diff
@@ -116,7 +121,8 @@ async def update_evaluation_row(evaluation_row_id: str, evaluation_row: Evaluati
         'outputs': evaluation_row_dict["outputs"]
     }
 
-    if evaluation_type == EvaluationType.auto_exact_match:
+    if (evaluation_type == EvaluationType.auto_exact_match or
```

Comment:
In general I'd suggest to always keep the logic of the app outside of the router (in the service instead). The router should only deal with the Exceptions / communication.

Reply:
But we can keep that for a refactor. In any case, this was done quick and dirty for now